public/javascripts/main.js

```diff
@@ -35,6 +35,10 @@ $(function(defaultDateStart, defaultDateEnd){
 
 $(document).ready(function() {
     allSelected();
+
+    $('button.btn-danger').click(function() {
```

Comment:
this selector is too generic, this would mean that every button with class `btn-danger` is going to show the confirmation message.
Please introduce a new selector on the button (something like `js-delete-confirmation`) and then update this line.